pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
     "pandas>=1.5.0",
     "scikit-learn>=1.1",
     "scipy>=1.8.1",
+    "sentence-transformers~=2.2.2",
```

Comment:
Not that we can do anything about it, but `sentence-transformers` also brings in `torchvision` as dependency, which is unnecessary for NLP / Text use-cases. 😢

Reply:
bummer.  I didn't notice that.  I thought we did good to avoid some other baggage.  :(


https://github.com/UKPLab/sentence-transformers/pull/1881